src/process_utility.c

```diff
@@ -2814,6 +2814,20 @@ process_altertable_start_matview(ProcessUtilityArgs *args)
 				process_altercontinuousagg_set_with(cagg, view_relid, (List *) cmd->def);
 				break;
 
+			case AT_ChangeOwner:
+				/* Change owner of view */
+				AlterTableInternal(view_relid, list_make1(cmd), false);
```

Comment:
This seems to share a lot of code with the `AT_SetTableSpace` case. Would be good to harmonize.

I also think we need to change ownership of the other objects, the direct view, partial view, etc.

Reply:
Fixed.